css/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0 auto;
+  padding-left: 5px;
+}
+.firstfield {
+  display: flex;
+  flex-wrap: wrap;
+}
+.firstfield > svg {
+  padding: 1px;
+}
+.kiwi-tag {
+  fill:orange;
+}
+.lake {
+  fill:blue;
+}
+.kiwi-object {
+  width: 150px;
+}
+.kiwi-base64 {
+  width: 150px;
+}
+/*for 3*/
+.circle-anim {
+  animation-duration: 2s;
+  animation-name: circleanim;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+  @keyframes circleanim {
+    from {
+      transform: rotate(0deg);
+    }
+    to { 
+      transform: rotate(360deg);
+    }
+  }
+/*for 4*/
+@media all and (min-width: 1800px) {
+  .svg-resp {
+    width: 800px;
+    height: 400px;
+  }
+}
+@media all and (max-width: 1800px) and (min-width: 1201px)  {
+  .svg-resp {
+    width: 600px;
+    height: 300px;
+  }
+}
+@media all and (max-width: 1200px) and (min-width: 961px)  {
+  .svg-resp {
+    width: 500px;
+    height: 250px;
+  }
+}
+@media all and (max-width: 960px) and (min-width: 769px)  {
+  .svg-resp {
+    width: 400px;
+    height: 200px;
+  }
+}
+/*for 5*/
+.container {
+  height: 100%;
+  position: relative;
+}
+.container__box {
+  margin: 0;
+  padding: 20px;
+  text-align: center;
+  color: #ffffff;
+  background: #9e54bd;
+  position: relative;
+  z-index: 5;
+}
+.svg {
+  cursor: pointer;
+}
+.btn {
+  background: none;
+  cursor: pointer;
+  padding: 25px 100px;
+  display: inline-block;
+  text-transform: uppercase;
+  font-size: 20px;
+  letter-spacing: 2px;
+  font-weight: 700;
+  border: 3px solid #fff;
+  color: #fff;        
+  position: relative;
+  transition: 0.5s;
+}
+.btn:after {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  transition: 0.5s;
+}
+.btn--next {
+  overflow: hidden;
+}
+.btn--next:after { /*можна використати transform: scale(500), задавши ширину 2п, висоту 1п, опасіті 0, а в :hover:after включити опасіті;*/
```

Comment:
Good that you left this comment, bad that it's not in English.

Reply:
Edited.